jmclient/jmclient/configure.py

```diff
@@ -262,6 +262,12 @@ def jm_single():
 #        2. You cannot change the type of a pre-existing wallet.
 native = true
 
+# In joinmarket-qt, display zpub for native segwit wallet, and ypub for
+# non-native segwit wallet.
+# If set to false, the extended public key will always display in the xpub
+# format.
+qt_display_ypub_zpub = true
```

Comment:
Not 100% sure default should be true, but probably. What do others think? @AdamISZ ? @chris-belcher ?